### DISCUSSION TITLE
Exclude subfolders from laravel-medialibrary

### DISCUSSION THREAD

**Discussion by lpeterke** [spatie/laravel-backup#1511] [Q&A]
I pretty much have the same question that has been asked here: https://github.com/spatie/laravel-backup/issues/1002

I am using the laravel-medialibrary package and I'd like to exclude all the "conversions" subfolders, as they can always be regenerated with an Artisan command. 

This way I could keep my backup size a lot smaller! Is this possible in some way? Or do I have to use this: https://spatie.be/docs/laravel-medialibrary/v10/advanced-usage/using-a-custom-directory-structure?

Cheers!

**Comment by erikn69** [comment]
Yes, it is possible, read documentation, on exclude files and directories
